extensions/amp-analytics/analytics-vars.md

```diff
@@ -473,4 +473,11 @@ Provides the total time from the time page was loaded to the time a ping was sen
 
 Provides the total time for which the element has met the visiblitySpec conditions at time this ping is sent.
 
+### fromSlide
```

Comment:
These should be in their own section. For that matter, the amp-access variables should also probably be here, too (also in their own section) (cc @dvoytenko)